shell.nix

```diff
@@ -1,4 +1,4 @@
-{ nixpkgs ? fetchTarball https://github.com/NixOS/nixpkgs/archive/7fff567ee99c1f343ecdd82fef2e35fb6f50e423.tar.gz
+{ nixpkgs ? fetchTarball https://github.com/NixOS/nixpkgs/archive/8d3e91077ba074e2c947a152ee8ab7be885c42ab.tar.gz
 , pkgs ? import nixpkgs {}
 }:
 
```

Comment:
while you are changing this file, can you fix the line below:

```nix
stdenv.mkDerivation {	
  name = "rust-carano";	

  # ...
}
```

Reply:
Done!